lib/Doctrine/Sniffs/Operators/NegationOperatorSpacingSniff.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Sniffs\Operators;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+use function in_array;
+use function strlen;
+use const T_CLOSE_PARENTHESIS;
+use const T_CLOSE_SHORT_ARRAY;
+use const T_CLOSE_SQUARE_BRACKET;
+use const T_CONSTANT_ENCAPSED_STRING;
+use const T_DNUMBER;
+use const T_ENCAPSED_AND_WHITESPACE;
+use const T_LNUMBER;
+use const T_MINUS;
+use const T_NUM_STRING;
+use const T_STRING;
+use const T_VARIABLE;
+use const T_WHITESPACE;
+
+final class NegationOperatorSpacingSniff implements Sniff
```

Comment:
Can we make it for both `+` and `-`?

Reply:
Nah, that doesn't make sense ... where would you use `$x = +$y` and why? If anything, I would ban such constructs.

Reply:
One can cast a string to an integer: https://3v4l.org/9JEfb.

I don't think we do that in Doctrine, but for sure others do in their project that uses our standards.

Reply:
Wow, didn't think about such use-case :D To be honest, I would rather not implement this, but if you think it should be there, I will look into it tomorrow.

Reply:
I think we can skip the use-case - as you said, I'd rather not allow such constructs in the first place.